src/main.py

```diff
@@ -19,6 +19,7 @@
 from src import mailer
 
 SENTRY_DSN = os.environ.get("SENTRY_DSN")
+ENVIRONMENT = os.environ.get("ENVIROMENT") or "dev"
```

Comment:
Wouldt this also work?
```suggestion
ENVIRONMENT = os.environ.get("ENVIROMENT", "dev")
```

Reply:
Yes it will. I'll fix. And I see that i misspelt ENVIRONMENT again. That darn N